rustls/src/common_state.rs

```diff
@@ -883,6 +905,17 @@ enum Limit {
     No,
 }
 
+pub(crate) enum RpkNegotiationResult {
+    Negotiated,
+    NotNegotiated,
+    Err(Error),
+}
+
+pub(super) struct RawKeyNegotationParams {
```

Comment:
ditto RE: `Debug`. I think it's almost universally helpful for new struct/enums.